sig-auth/README.md

```diff
@@ -37,6 +37,73 @@ The Chairs of the SIG run operations and processes governing the SIG.
 * [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-auth)
 * [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2Fauth)
 
+## Subprojects
+
+The following subprojects are owned by sig-auth:
+- **audit-logging**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes/kubernetes/master/staging/src/k8s.io/apiserver/pkg/apis/audit/OWNERS
+    - https://raw.githubusercontent.com/kubernetes/kubernetes/master/staging/src/k8s.io/apiserver/pkg/audit/OWNERS
```

Comment:
the raw links are harder to view on mobile devices… any strong reason to prefer them over the web-viewable links?

Reply:
it's not the greatest justification but the goal is for automation to be able to pull these files down and consume them, and it seemed easier to link directly to the file than provide a github ui link

Reply:
Ah, ok